README.md

```diff
@@ -40,31 +40,33 @@ Your _**application.conf**_ should be configured to enable clamav scanning
 
 ```JavaScript
 clam.antivirus {            
-    enabled = true          
-    chunkSize = 32768       
-    host = avscan           
+    host = avscan
     port = 3310             
     timeout = 5000          
-    threadPoolSize = 20     
-    maxLength = 10485760    
 }
 ```
 
-Wire up your microservice to load the ClamAvConfig
+Wire up your microservice to load the ClamAvConfig using dependency injection within your Module. For an explanation on using Guice dependency injection within your Play project, see the [Play documentation](https://www.playframework.com/documentation/2.5.x/ScalaDependencyInjection).
 
 ```JavaScript
-object ClamAvConfiguration extends RunMode {
-
-  lazy val config: Option[Configuration]  = Play.current.configuration.getConfig(s"$env.clam.antivirus")
-  lazy val clamAvConfig = LoadClamAvConfig(config)
-
+class YourModule extends Module {
```

Comment:
Maybe it's worth creating our own Guice module with antivirus. (currently containing only eager binding to clamav config)
Then, if we abstract structure of the service from users. So that if we add/remove components to it, it won't affect clients?